addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -25,7 +25,7 @@
 </template>
 
 <template id="s_newsletter_subscribe_form" name="Newsletter">
-    <div class="s_newsletter_subscribe_form js_subscribe" data-vxml="001" data-list-id="0">
+    <div class="s_newsletter_subscribe_form js_subscribe" data-vxml="001" data-list-id="0" data-name="Newsletter Form">
```

Comment:
Should not be necessary, it should get the name of the view automatically (which hopefully will be translated some day).

Reply:
Sadly it is not, or I wouldn't have added it. Maybe it has to do with the fact is is `t-snippet-call`'d
See without it (notice the "Block" in right panel
![image](https://user-images.githubusercontent.com/30048408/128872856-ee55c835-4499-4bfb-85a8-91d0364c1139.png)

And with this diff
![image](https://user-images.githubusercontent.com/30048408/128872805-266d6252-3309-445f-96df-9a6fa9b4a977.png)

Reply:
Indeed... we should fix t-snippet-call instead